src/Bridge/Symfony/DependencyInjection/Configuration.php

```diff
@@ -31,8 +33,23 @@ public function getConfigTreeBuilder(): TreeBuilder
 
         $rootNode
             ->children()
-                ->scalarNode('api_key')->end()
-                ->scalarNode('api_url')->end()
+                ->scalarNode('api_key')
+                    ->isRequired()
+                    ->info('Gandi api key')
+                ->end()
+                ->scalarNode('api_url')
+                    ->defaultValue(Gandi::DEFAULT_API_URL)
+                    ->info('Gandi remote api endpoint.')
+                ->end()
+                ->scalarNode('xml_rpc_client')
+                    ->defaultValue('nexy_gandi.default_xml_client_rpc')
```

Comment:
Why would you need to change the RPC client?

Reply:
Because this give hand on Client transport without having to depend on magic find methods `new HttpAdapterTransport(MessageFactoryDiscovery::find(), HttpClientDiscovery::find())`

Reply:
I'm not sure to understand. We don't do magic find method, this depend to `lstrojny/fxmlrpc`.

Reply:
yes this depend to `lstrojny/fxmlrpc` but it construct also give possibility to change transport, this is not possible without having to create a compiler pass to inject custom XML RPC client right now. Having to depend on autodiscover can introduce weird bugs after requiring a new dependence in project.

Reply:
Ok then.